### PR TITLE
Fixes #26354 - Fix duplicate installable hosts count

### PR DESCRIPTION
**Pull request by hao-yu**

### CODE REVIEW

**Comment by jturel** [comment]
[test katello]

File: app/models/katello/erratum.rb [hunk 1]
```diff
@@ -86,7 +86,7 @@ def hosts_applicable(org_id = nil)
     end
 
     def hosts_available(org_id = nil)
-      self.hosts_applicable(org_id).joins("INNER JOIN #{Katello::RepositoryErratum.table_name} on \
+      self.hosts_applicable(org_id).distinct.joins("INNER JOIN #{Katello::RepositoryErratum.table_name} on \
```
Comment (jturel) [inline app/models/katello/erratum.rb]: @hao-yu would you be comfortable adding a test for this?

Comment (hao-yu) [reply app/models/katello/erratum.rb]: Yup, I can do that

Comment (hao-yu) [reply app/models/katello/erratum.rb]: Somehow the pull request not getting updated. I will submit a new one.

**Comment by hao-yu** [comment]
Somehow the pull request not getting updated. I will submit a new one.

**Pull request closed without merging**